### PR TITLE
PIM-6537: fix role page look

### PR DESCRIPTION
**Pull request by juliensnz**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/UserBundle/Resources/views/Role/update.html.twig [hunk 1]
```diff
@@ -220,56 +216,5 @@
         </div>
 
     {{ form_end(form) }}
-
```
Comment (tamarasaurus) [inline src/Pim/Bundle/UserBundle/Resources/views/Role/update.html.twig]: this wasn't used?

Comment (juliensnz) [reply src/Pim/Bundle/UserBundle/Resources/views/Role/update.html.twig]: nope ^^

File: src/Pim/Bundle/UIBundle/Resources/public/less/components/product-edit-form/VerticalNavtab.less [hunk 2]
```diff
@@ -1,29 +1,24 @@
...
+    border-right: 1px solid @AknBorderColor;
+    border-bottom: 1px solid @AknBorderColor;
   }
 
   &-link {
     height: @AknTabHeight;
     line-height: @AknTabHeight;
     color: @AknDefaultFontColor;
-    background: lighten(@AknGrey, 30%);
+    background: white;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/less/components/product-edit-form/VerticalNavtab.less]: no var for that ?

File: src/Pim/Bundle/UIBundle/Resources/public/less/components/product-edit-form/VerticalNavtab.less [hunk 3]
```diff
@@ -39,18 +34,18 @@
 
   &-link--active,
   &-item--active &-link {
-    background: lighten(@AknGrey, 20%);
+    background: white;
     color: @AknDarkFontColor;
-    font-weight: bold;
+    font-size: 13px;
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/less/components/product-edit-form/VerticalNavtab.less]: Please use 12/14 and use the variable.less variable

**Pull request merged**